--- xtel-3.3.0.orig/XTelm.ad
+++ xtel-3.3.0/XTelm.ad
@@ -10,7 +10,7 @@
 
 XTelm*serveur:			localhost
 XTelm*commandeImpression:	pnmflip -r90 %s | pnmtops | lpr -h
-XTelm*commandeImpressionAscii:	a2ps %s | lpr -h
+XTelm*commandeImpressionAscii:	a2ps -o - %s | lpr -h
 XTelm*petiteFonte:		False
 XTelm*nomService:		xtel
 ! Type d'arret: 0=Pas d'arret, 1=Form Feed, 2=SUITE, 3=SUITE ou Form Feed
--- xtel-3.3.0.orig/XTel.ad
+++ xtel-3.3.0/XTel.ad
@@ -10,7 +10,7 @@
 
 XTel*serveur:			localhost
 XTel*commandeImpression:	pnmflip -r90 %s | pnmtops | lpr -h
-XTel*commandeImpressionAscii:	a2ps %s | lpr -h
+XTel*commandeImpressionAscii:	a2ps -o - %s | lpr -h
 XTel*petiteFonte:		False
 XTel*nomService:		xtel
 XTel*font:			-*-helvetica-medium-r-*-*-14-*-*-*-*-*-iso8859-1
